onnxruntime/core/graph/graph_utils.h

```diff
@@ -49,8 +65,9 @@ bool GetRepeatedNodeAttributeValues(const Node& node,
 Status ForAllMutableSubgraphs(Graph& main_graph, std::function<Status(Graph&)> func);
 Status ForAllSubgraphs(const Graph& main_graph, std::function<Status(const Graph&)> func);
 
-/** Remove the given single-input-single-output Node from the Graph. */
-bool RemoveSingleInSingleOutNode(Graph& graph, Node& node);
+/** Remove the given single-input Node from the Graph. The single input might be either
+    another node or an initializer.*/
+bool RemoveSingleInputNode(Graph& graph, Node& node);
 
```

Comment:
Is there a unit test that validates the removal of multiple output edges?

Reply:
We should add one. I am not sure if any of the graph transform test models we have tests that, but I think we should have a more explicit test. I'll add one.

Reply:
Done with latest commit.